src/core/styles/components/drop-down/_drop-down-theme.scss

```diff
@@ -0,0 +1,260 @@
+        $hover-item-text-color: igx-color($palette, 'grays', 800),
+
+        $hover-item-background: igx-color($palette, 'grays', 100),
+
+        $focused-item-background: igx-color($palette, 'grays', 100),
+        $focus-item-text-color: igx-color($palette, 'grays', 800),
+
+        $selected-item-background: igx-color($palette, 'secondary', 500),
+        $selected-item-text-color: #fff,
+
+        $disabled-item-background: transparent,
+        $disabled-item-text-color: igx-color($palette, 'grays', 500)
+    );
+
+    $this: extend($default-theme, $theme);
+
+    $desktop-item-height: 32px;
+    $mobile-item-height: 48px;
+
+    $desktop-item-min-width: 128px;
+    $mobile-item-min-width: 112px;
+
+    $desktop-item-padding: 24px;
+    $mobile-item-padding: 16px;
+
+    $desktop-header-padding: 16px;
+    $mobile-header-padding: 8px;
+
+    $mobile-break-point: 600px;
+
+
+    $dd-shadow: igx-elevation($elevations, 3);
+
+    %igx-drop-down {
+        // We need that calc to fix a bug with ellipsis which makes the ellipsis to appear even when there is no need
+        max-width: calc(100% - 32px);
+        min-width: rem($desktop-item-min-width);
+        max-height: 100%;
+        position: absolute;
+        @media only screen and (max-width: $mobile-break-point) {
+            min-width: rem($mobile-item-min-width);
+        }
+    }
+
+    %igx-drop-down__list {
+        overflow-y: auto;
+        z-index: 1;
+        // We need that calc to fix a bug with ellipsis which makes the ellipsis to appear even when there is no need
+        min-width: calc(100% + 32px);
+        box-shadow: $dd-shadow;
+        background: map-get($this, 'drop-down-background-color');
+    }
+
+    %igx-drop-down__header,
+    %igx-drop-down__item {
+        display: flex;
+        justify-content: flex-start;
+        align-items: center;
+        width: 100%;
+        height: rem($desktop-item-height);
+        line-height: rem($desktop-item-height);
+        min-height: rem($desktop-item-height);
+        white-space: nowrap;
+
+        @media only screen and (max-width: $mobile-break-point) {
+            height: rem($mobile-item-height);
+            line-height: rem($mobile-item-height);
+            min-height: rem($mobile-item-height);
+        }
+    }
+
+    %igx-drop-down__item {
+        color: map-get($this, 'item-text-color');
+        cursor: pointer;
+        padding: 0 rem($desktop-item-padding);
+        @media only screen and (max-width: $mobile-break-point) {
+            padding: 0 rem($mobile-item-padding);
+        }
+
+        &:focus {
+            outline: 0;
+            outline-color: transparent;
+            background: map-get($this, 'focused-item-background');
+            color: map-get($this, 'focus-item-text-color');
+        }
+
+        &:hover {
+            background: map-get($this, 'hover-item-background');
+            color: map-get($this, 'hover-item-text-color');
+        }
+    }
+
+    [dir='rtl']{
```

Comment:
This doesn't work if the direction is set on a parent item. RTL should work even if the direction is set on parent items.